lib/rules/web/admin_console/4.6/overview.xyaml

```diff
@@ -216,4 +226,65 @@ group_by_label:
     dropdown_menu_item: <label>
   action: click_groupby_dropdown_button
   action: click_dropdown_menu_item
-
+click_workloads_tab:
+  element:
+    selector:
+      xpath: //a[contains(text(),'Workloads')]
+    op: click
+send_key_to_workloads:
+  element:
+    selector:
+      xpath: //a[contains(text(),'Workloads')]
+    op: send_keys <key>
+check_workload_selected:
+  elements:
+  - selector:
+      xpath: //li[contains(@class,'pf-c-data-list__item') and contains(@class,'pf-m-selected')]
```

Comment:
Here, should check the expected workload is selected, eg: $x("//li[contains(@class, 'pf-m-selected')]//div[contains(., '<workload_name>')]"). WDYT?

Reply:
yea but because of the backward compatibility of the versions it has to be omitted.